easycv/models/classification/classification.py

```diff
@@ -110,16 +110,29 @@ def __init__(self,
         self.init_weights()
 
     def init_weights(self):
+        logger = get_root_logger()
         if isinstance(self.pretrained, str):
-            logger = get_root_logger()
             load_checkpoint(
                 self.backbone, self.pretrained, strict=False, logger=logger)
-        else:
-            print_log('load model from init weights')
+        elif self.pretrained:
             if self.backbone.__class__.__name__ == 'PytorchImageModelWrapper':
                 self.backbone.init_weights(pretrained=self.pretrained)
+            elif hasattr(self.backbone,
+                         'pretrained_path') and self.backbone.pretrained_path:
```

Comment:
why backbone has the paramerater  `pretrained_path`,  what's the difference with param `pretrained`

Reply:
got it, may be a better name helps understanding,  default_pretrained_model_path